### DISCUSSION TITLE
Is it normal for traffic outside the container to be routed through the VPN?

### DISCUSSION THREAD

**Discussion by nic-kill** [haugene/docker-transmission-openvpn#2336] [Q&A]
To be explicit the containers are running fine and as expected behind the VPN.

My question is if all traffic on the computer should be getting routed through the VPN? When i do:

`curl icanhazip.com`

the displayed IP address is the VPN endpoint, both inside the container and outside it. I'm not sure if i am misunderstanding how VPNs work in containers or if this is not normal behaviour.

**Comment by pkishino** [comment]
Please provide more details.. what network mode are you using in docker? You mention other containers, what config are you using? 
In a “usual” setup, only your transmission container is routed through the vpn, nothing else.. but some people route all their containers through this one or even all their traffic, it all depends on their setup
